lib/rubocop/cop/rails/find_each.rb

```diff
@@ -39,7 +39,13 @@ def autocorrect(node)
         private
 
         def method_chain(node)
-          [*node.ancestors, node].map(&:method_name)
+          chain = []
+          loop do
+            chain << node.method_name
+            node = node.receiver
+            break unless node.send_type?
+          end
```

Comment:
I think `Node#descendants` is better in this case :)

Reply:
Right! it looks like simply changing  `ancestors` by `descendants` on my first implementation will work. I've pushed another commit. Thanks!